examples/index.html

```diff
@@ -64,10 +41,11 @@ <h3 class="font-size-small">In Text:</h3><br/>
     </div>
     <!-- END OF EXAMPLE -->
   </body>
-  <script src="../node_modules/leaflet/dist/leaflet.js"></script>
-  <script src="../node_modules/jquery/dist/jquery.min.js"></script>
-  <script src="../src/locationForm.js"></script>
-  <script src="../src/location_tags.js"></script>
-  <script src="../src/main.js"></script>
-  <script src="../src/object.js"></script>
+
+  <script>
+    var blurredLocation = new BlurredLocation();
+    panMapToGeocodedLocation('geo_location');
```

Comment:
Here, let's use this as an internal method to blurredLocation -- like `blurredLocation.panMapToGeocodedLocation(...)` -- i'll show you how, below. Same with `panMapWhenInputsChange`.